#lidar_port 
port: "/dev/ttyACM1"
#USB camera port selection
video_device : "/dev/video0"
#camera and lidar saved images directory
Cam_lidar_images_directory : "/home/mir-lab/Desktop/"
#stacked images saved images directory
stacked_images_directory : "/home/mir-lab/Desktop/"
